### ISSUE TITLE
how to go about making the GUISANDBOX not pop up when running room_camera_intersection

### ISSUE THREAD

**Issue by juanhotencoding** [jmccormac/pySceneNetRGBD#24]
Hello,

I was wondering if there was a quick and easy way of making the GUISANDBOX not pop up when running the camera trajectory code. Would it just be commenting out some lines of code? I am running on a server that throws an error with X11 display.

**Comment by arjunds** [comment]
The camera trajectory visualizer is all written in Pangolin and, theoretically, commenting out line 475:
`pangolin::CreateWindowAndBind("GUISandbox",w_width+150,w_height);`
in camera_trajectory_generator.cpp should stop the GUISandbox from showing up (but it will still render the frames and do all the visualization steps except without a visible GUI). If this doesn't work, or you want to speed it up by completely removing all the visualization code, search for lines that reference pangolin, or the following variables `d_panel, s_cam, saved_s_cam, d_cam`
Hope this helps!

**Comment by juanhotencoding** [comment]
Just commenting out line 475 in the camera_trajectory_generator.cpp did not work, it Seg Faulted, looks like I will have to, as you mentioned, comb through the code for all instances of pangolin.

**Comment by arjunds** [comment]
Yeah sorry you'd have to comment out everything just because it tries to add panels to the GUI but there is no GUI after commenting out line 475 - It makes sense that it would seg fault.

**Comment by BKAUTO** [comment]
@juanhotencoding , Hello! I ran into the same problem with you when I am trying to disable the GuiSanbox. Have you solved the problem? Could you please tell me which lines need to be modified in camera trajectory codes?
